### PR TITLE
Starting assignment

### PR DESCRIPTION
**Pull request by ShaunWMusic**

### CODE REVIEW

**Comment by ShaunWMusic** [comment]
- [x] Create a new project using `yo sass-broccoli` name the project `06-tshirts`
- [x] Initialize new folder as git repository
- [x] Using `hub` create a new repository on github
- [x] Checkpoint: Commit code from HTML grouping workshop to `master branch`
- [x] Make one change to project and commit changes
- [x] Push changes to `origin` `develop` branch
- [x] Create a pull request using the `hub` CLI
- [x] **GOAL**: Style Grid Basics
  - [x] Outer container gutter padding
  - [x] Inner container max width
  - [x] Stack Grid Items Horizontally
- [x] **GOAL**: Style Shop Item
  - [x] Style: Border around Shop Item
  - [x] Style: Image Gutter
  - [x] Style: Button Row
  - [x] Style: Button Border
  - [x] Style: Drop Down Position
  - [x] Style: Drop Down Stack
  - [x] Style: Drop Down Item Border
  - [x] Style: Button Hover State
  - [x] Style: Dropdown Hover State
- [x] **GOAL**: Style Color Icons
  - [x] Style: Pink Box
  - [x] Style: Blue Box
  - [x] Style: Yellow Box
  - [x] Style: Green Box
- [x] **GOAL**: Grid Settings Phone
  - [x] Style: Grid Wrap
  - [x] Style: Grid Min Width
- [x] **GOAL**: Tablet Layout
  - [x] Style: Full Width
  - [x] Style: Shop Item Changes
  - [x] Style: Buttons
  - [x] Style: Dropdown Position
  - [x] Style: Dropdown Changes

**Comment by rtablada** [comment]
There are some double borders around the action buttons:

screen shot 2016-10-12 at 2 33 32 pm

The grid is not being centered on desktop sizes: 

screen shot 2016-10-12 at 2 33 42 pm

The hoverstates are a bit strange. Right now it looks like the entire button is being scaled rather than the inner contents.

screen shot 2016-10-12 at 2 34 01 pm

File: styles/app.scss [hunk 1]
```diff
@@ -1,8 +1,184 @@
...
+  }
+  .shop-menu__tooltip-item {
+    border-right: 0;
+    // padding: 0 .5rem;
+
+  }
+}
+
+//borders for desktop
+@media(min-width: 779px) {
```
Comment (rtablada) [inline styles/app.scss]: Also set a max-width so this only applies to tablet sizes

**Review by rtablada — COMMENTED** [review]
Just look at the media query so that the changes show on tablet, but not mobile.

File: styles/app.scss [hunk 2]
```diff
@@ -1,8 +1,184 @@
...
 
 $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
+@import 'resets';
 
-@import 'adonis';
+.container {
+  max-width: 93rem;
+  margin: $gutter;
```
Comment (rtablada) [inline styles/app.scss]: Check the margin on containers to center the items in large screens.

**Pull request merged**